sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -293,14 +297,15 @@ export interface KeyReleasePolicy {
 }
 
 // @public
-export type KeyType = string;
+type KeyType_2 = string;
+export { KeyType_2 as KeyType }
```

Comment:
this has to be some kind of bug, right?

Reply:
https://github.com/microsoft/TypeScript/blob/b2f9432c4064c68862f992b750d7b40c64f194b4/lib/lib.dom.d.ts#L633

Reply:
Speaking of differences - KeyType for us does not mean the same as keyType in node crypto - for us it's the `kty` (EC, RSA, etc). I _think_ globally it's defined as "private" | "public" | "secret"